docroot/modules/humsci/hs_capx/src/Overrides/ConfigOverrides.php

```diff
@@ -205,4 +222,17 @@ public function createConfigObject($name, $collection = StorageInterface::DEFAUL
     return NULL;
   }
 
+  /**
+   * Avoid circular dependency reference when using dependency injection.
+   *
+   * @return \Psr\Log\LoggerInterface
+   */
+  protected function getLogger(): LoggerInterface {
```

Comment:
I don't know if I've seen this before. What's the issue that comes up?

Reply:
It's been a couple weeks since I wrote this, so my memory is a bit fuzzy, but there was some sort of circular dependency when using dependency injection that required config to already exist, but this method is all about setting up config.